cmd/liquidator.go

```diff
@@ -0,0 +1,138 @@
+package cmd
```

Comment:
I would pull this entire file into a package at the root called `liquidator`. It doesn't really belong in the `cmd` pkg IMO.

Reply:
done at [b1afef4](https://github.com/umee-network/liquidator/pull/1/commits/b1afef47e55513f5a26513593e6b06a98492153f)